test/javascripts/acceptance/question-answer-test.js

```diff
@@ -568,7 +574,7 @@ acceptance("Discourse Question Answer - logged in user", function (needs) {
   // Skip message bus tests but keep it around for development use. We currently do not have a reliable way to wait for
   // widgets to re-render and this results in all kinds of timing issues. For example, this test passes in the browser
   // but fails when ran in headless mode.
-  skip("receiving user post voted message where current user removed their vote", async function (assert) {
```

Comment:
I learnt about `await settled()` today which resolved the problem we were seeing previously.